### DISCUSSION TITLE
example of subscription?

### DISCUSSION THREAD

**Discussion by tigerinus** [graphql/graphql-js#3637] [Q&A] [answered]
Hello,

I don't find any doc about implementing subscription at server side. Any pointers?

Thanks!

T.

**Comment by saihaj** [comment, marked as answer]
You can checkout these docs https://www.graphql-yoga.com/docs/features/subscriptions

**Reply by tigerinus** [reply]
`graphql-yoga` is actually the library I am using right now. I wanted to do a comparison between `graphql-js` and `graphql-yoga`.

So....is `graphql-yoga` the successor of `graphql-js` or does it depend on `graphql-js`?

**Reply by saihaj** [reply]
`graphql-yoga` depends on `graphql-js`. `graphql-js` is a building block. With `graphql-yoga` you get a full transport layer using server sent events.
